tests/e2e/e2e_test.go

```diff
@@ -37,6 +37,8 @@ import (
 
 // TODO: Find more scalable way to do this
 func (s *IntegrationTestSuite) TestAllE2E() {
+	t := s.T()
+	t.Parallel()
```

Comment:
learning that parallelizing e2e did not work (breaks the tests) and that the calls to t.Parallel() in the subtests don't seem to do anything